api/src/client/dynamics/housing/DynamicsHousingRegistrationStatusClient.ts

```diff
@@ -1,24 +1,29 @@
 import {
-  HotelMotelRegistrationClient,
   HousingPropertyInterestClient,
+  HousingRegistrationClient,
   HousingRegistrationRequestLookupResponse,
 } from "@client/dynamics/housing/types";
 import { AccessTokenClient } from "@client/dynamics/types";
 import { LogWriterType } from "@libs/logWriter";
 
 type Config = {
   accessTokenClient: AccessTokenClient;
-  housingHotelMotelRegistrationClient: HotelMotelRegistrationClient;
+  housingHotelMotelRegistrationClient: HousingRegistrationClient;
```

Comment:
Noticed we updated the properties in the housingRouter.ts file. We should be updating this as well `housingHotelMotelRegistrationClient`